lib/embed-docs/embedDocs.ts

```diff
@@ -279,3 +252,77 @@ export function splitTextByHeaders(
 
   return results;
 }
+
+export function splitIntoTextChunks(
+  sectionText: string,
+  ignoreLines: string[],
+): string[] {
+  return joinShortChunks(
+    sectionText
+      .split("\n")
+      .filter((chunk) => isTextWithSubstance(chunk))
```

Comment:
why do you need this and the rejoining of short chunks?

Reply:
So sometimes there are not-useful lines with something like `[` on a line on its own. This doesn't add anything.

Neither does the date at the bottom of the page, or a url to the privacy page or... you get the picture. We want the lines we include to have some substance to them

Reply:
Added a comment